integrations/src/test/resources/io/jenkins/plugins/casc/GitTest.yml

```diff
@@ -0,0 +1,17 @@
+unclassified:
+  globalLibraries:
+    libraries:
+      - defaultVersion: "1.2.3"
+        name: "Test Git Lib"
+        retriever:
+          legacySCM:
+            scm:
+              git:
+                branches:
+                  - name: "*/myprodbranch"
+                browser:
+                  # Reproduces JENKINS-57604
+                  assemblaWeb:
```

Comment:
@oleg-nenashev did you check if this works for all kind of browsers?

Reply:
Nope, but it works for few browsers I tested. I am addressing the issue on the JCasC side at the moment, full browser compatibility is rather a subject for test automation on the Git plugin's side. If it helps, I can add few extra browsers here

Reply:
No need, I just wanted to confirm it works for other browsers. Thanks!